CMakeLists.txt

```diff
@@ -141,7 +141,8 @@ endif()
 # Build options
 #------------------------------------------------------------------------------
 option(BUILD_SHARED_LIBS "Build with shared libraries." ON)
-option(HERMES_INTERCEPT_IO "Use GOTCHA as IO interception tool" ON)
+option(BUILD_TESTING "Build testing." ON)
```

Comment:
According to [the documentation](https://cmake.org/cmake/help/v3.10/module/CTest.html?highlight=build_testing), the `CTest` module "automatically creates a `BUILD_TESTING` option that selects whether to enable testing support (`ON` by default)."

Reply:
Yes, I was thinking to rename it to `HERMES_BUILD_TESTING`, then there will be no confusion.